src/Provider/GoogleMaps/Model/GoogleAddress.php

```diff
@@ -103,7 +158,197 @@ public function withFormattedAddress(string $formattedAddress = null)
     /**
      * @return null|string
      */
-    public function getSubpremise()
+    public function getAirport(): string
+    {
+        return $this->airport;
+    }
+
+    /**
+     * @param null|string $airport
+     */
+    public function withAirport(string $airport = null)
+    {
+        $new = clone $this;
+        $new->airport = $airport;
+
+        return $new;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getColloquialArea(): string
+    {
+        return $this->colloquialArea;
+    }
+
+    /**
+     * @param null|string $colloquialArea
+     */
+    public function withColloquialArea(string $colloquialArea = null)
+    {
+        $new = clone $this;
+        $new->colloquialArea = $colloquialArea;
+
+        return $new;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getIntersection(): string
+    {
+        return $this->intersection;
+    }
+
+    /**
+     * @param null|string $intersection
+     */
+    public function withIntersection(string $intersection = null)
+    {
+        $new = clone $this;
+        $new->intersection = $intersection;
+
+        return $new;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getNaturalFeature(): string
+    {
+        return $this->naturalFeature;
+    }
+
+    /**
+     * @param null|string $naturalFeature
+     */
+    public function withNaturalFeature(string $naturalFeature = null)
+    {
+        $new = clone $this;
+        $new->naturalFeature = $naturalFeature;
+
+        return $new;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getNeighborhood(): string
+    {
+        return $this->neighborhood;
+    }
+
+    /**
+     * @param null|string $neighborhood
+     */
+    public function withNeighborhood(string $neighborhood = null)
+    {
+        $new = clone $this;
+        $new->neighborhood = $neighborhood;
+
+        return $new;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPark(): string
```

Comment:
We cannot use `:string` here since "park" might return null. Same for all others.